src/Services/organizationService.js

```diff
@@ -0,0 +1,53 @@
+import axios from 'axios';
+
+const getOrganizacion = async () => {
+  try {
+    const response = await axios(process.env.REACT_APP_GET_ORGANIZATION);
+
+    return response.data;
+  } catch (err) {
+    return err.respose.data;
+  }
+};
+
+const getOrganizacionById = async (id) => {
```

Comment:
this should be directly getOrganization. If there is no id it return all the organizations
```
const getOrganization = async (id?) => {
   try{
       const url = id ? `${process.env.GET_ORGANIZATION}/${id}` : `${process.env.GET_ORGANIZATION}`
       return await axios(url);  
   } catch (err) {
       return err.response.data;
   }
}
```